Libraries/test/EventsTests.Shared/EventTests.cs

```diff
@@ -482,6 +482,23 @@ public void DynamoDbUpdateTest(Type serializerType)
             var recordDateTime = record.Dynamodb.ApproximateCreationDateTime;
             Assert.Equal(recordDateTime.Ticks, 636162388200000000);
 
+            var topLevelList = record.Dynamodb.NewImage["misc1"].L;
+            Assert.Equal(0, topLevelList.Count);
+
+            var nestedMap = record.Dynamodb.NewImage["misc2"].M;
+            Assert.NotNull(nestedMap);
```

Comment:
Since we are doing this work to make properties not set in the event JSON are null on the POCO can you add some asserts to confirm they are null.

Reply:
Added a few asserts in https://github.com/aws/aws-lambda-dotnet/commit/3bd10bc4900e4d21ebba4d47fe9700cd0076a74a#diff-3eb0f1f2aaf46a4d65fa620dd99f96f1fba13a909a06a2f1e43c125cb6e1112e